src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -222,6 +220,26 @@ private LogQuery buildQuery(String offset, AdminActivityLogPageData data) {
         }
         
         String status="&nbsp;&nbsp;Total Logs gone through in last search: " + totalLogsSearched + "<br>";
+        
+        if (earliestLogChecked != null) {
+            String userGoogleId = earliestLogChecked.getId();
+            String userRole = earliestLogChecked.getRole();
+            
+            SimpleDateFormat sdf = new SimpleDateFormat("dd-MM-yy HH:mm:ss");
+            sdf.setTimeZone(TimeZone.getTimeZone(Const.SystemParams.ADMIN_TIME_ZONE));
+            
+            TimeZone adminTimeZone = TimeZone.getTimeZone(Const.SystemParams.ADMIN_TIME_ZONE);
+            Calendar adminTimezoneCalendar = Calendar.getInstance(adminTimeZone);
+            adminTimezoneCalendar.setTimeInMillis(earliestLogChecked.getTime());
+            
+            String timeInAdminTimeZone = sdf.format(adminTimezoneCalendar.getTime());
+            String timeInUserTimeZone = getLocalTimeInfo(userGoogleId, userRole, String.valueOf(earliestLogChecked.getTime()));
+            
+            status += "The earliest log entry checked on <b>" + timeInAdminTimeZone + "</b> in Admin Time Zone and on <b>" + timeInUserTimeZone + "</b> in Local Time Zone.<br>";
```

Comment:
Meaning if the search is bounded by a specified interval of time, we always return the lower bound? Otherwise return the earliest log entry's time?